docs/source/reference/graphos-features.mdx

```diff
@@ -18,7 +18,7 @@ For details on these features, see [this blog post](https://blog.apollographql.c
 
 <Tip>
 
-Try the Enterprise features of GraphOS Router with a free [GraphOS trial](https://www.apollographql.com/pricing).
+Try the Enterprise features of GraphOS Router with a [Free plan](https://www.apollographql.com/pricing).
```

Comment:
Need to actually check if this is true

Reply:
It appears the new free plan supports all router abilities https://docs.google.com/spreadsheets/d/1c9Kdr9Po7i6mLYPQwMYxhw2-_4TKbagoImifhRp5uKE/edit?gid=0#gid=0